test/spec.js

```diff
@@ -751,7 +754,7 @@ describe('checkDependencies', () => {
             consoleLogStub.restore();
             consoleErrorStub.restore();
 
-            Reflect.deleteProperty(process, 'exitCode');
+            delete process.exitCode;
```

Comment:
Whatever eslint says, `delete` is by far not equal to `Reflect.deleteProperty`, it is equavialent to
```js
if (Reflect.deleteProperty(obj, 'key')) throw new TypeError
```